frontend/app/src/composables/filters/transactions.ts

```diff
@@ -9,84 +10,102 @@ enum TransactionFilterKeys {
   START = 'start',
   END = 'end',
   ASSET = 'asset',
-  PROTOCOL = 'protocol'
+  PROTOCOL = 'protocol',
+  EVENT_TYPE = 'event_type'
 }
 
 enum TransactionFilterValueKeys {
   START = 'fromTimestamp',
   END = 'toTimestamp',
   ASSET = 'asset',
-  PROTOCOL = 'protocols'
+  PROTOCOL = 'protocols',
+  EVENT_TYPE = 'eventTypes'
 }
 
-type Filters = MatchedKeyword<TransactionFilterValueKeys>;
+export type Filters = MatchedKeyword<TransactionFilterValueKeys>;
```

Comment:
Hmm, I originally wanted to have these types completely self-contained and not exposed outside the composable.

Is there any specific reason why we can't use the `updateFilter` method to achieve the same result without modifying the code here at all, or at least keeping modifications to a minimum, by only passing a boolean to allow for disabling the protocol filter?